documentation/ame-guide/modules/ROOT/pages/generate/generate-openapi-doc.adoc

```diff
@@ -2,22 +2,23 @@
 
 To generate an OpenAPI specification from the Aspect Model, proceed as follows:
 
-* Click the *Generate* on xref:getting-started/ui-overview.adoc#menu-generate[menu]. +
-A menu offers one option to generate the OpenAPI specification. +:
-
-** *OpenAPI specification* -- continue with this one
+* Click the xref:getting-started/ui-overview.adoc#menu-generate[Generate menu] and select *OpenAPI Specification*.
 +
-image::generate-api-spec.png[Generate document, width=100%, align="left"]
+image:generate-api-spec.png[Generate document, width=15%]
+
 * In case your model supports multiple languages, select the specific language for the documentation.
 * Select *JSON* or *YAML* as the desired file format.
 * Set the *Base URL* were the Aspect API will be served from.
 * Check the *IncludeQueryAPI* checkbox, if you need a path section for the Query API Endpoint of the Aspect API to be included in the specification.
```

Comment:
Check checkbox .. Hmmm. I would prefer select or activate; let's record that in our rules (that we are consistent in all our docs).